elastic/kibana
# [SecuritySolution] override timerange for prebuilt templates

## Summary

Problem:
The timerange of Elastic prebuilt templates are fixed, plus they are immutable from UI, so users cannot change them when needed. It's not useful so we are fixing it in this PR.

Solution is to override the timerange when installing the templates to 24 hours ago to now, so when users duplicate the template,
they will get the same timerange.

Also when user clicking on the prebuilt timelines, we override it again to 24 hours ago to now.

Steps to verify:

1. Go to `kibana/x-pack/plugins/security_solution/server/lib/detection_engine/scripts`
2. sh ./timelines/delete_all_timelines.sh to delete all the timelines and templates
3. Landing on timelines page, templates tab, and click on each prebuilt templates.
4. Checking if the timerange is set to 24hours ago to now in the datepicker in flyout.
5. Try duplicate a prebuilt template, and make sure the time range you have is 7days ago to now.

### Checklist

Delete any items that are not applicable to this PR.
